ios/Yester/Info.plist

```diff
@@ -52,5 +51,22 @@
 			</dict>
 		</dict>
 	</dict>
+	<key>UIAppFonts</key>
```

Comment:
I think this config is not needed any more right?

Reply:
I don't think so. I uninstalled already both Icons packages, but can't find how to deinstall this UIAppFonts.